client/benches/tps/ci.rs

```diff
@@ -0,0 +1,22 @@
+//! One-shot benchmark by counting the CPU instructions and memory accesses
+//! using [iai](https://github.com/bheisler/iai)
+//! for checking performance improvements or regressions in CI environments
```

Comment:
Again. Running this kind of benchmark on the CI is going to be very unreliable. You want to run this on a stand that has repeatable performance. This is not the description of our AWS runners.

Reply:
Is it really useless even though [the doc](https://github.com/bheisler/iai#comparison-with-criterion-rs) says:

> Pro: Iai can work reliably in noisy CI environments or even cloud CI providers like GitHub Actions or Travis-CI, where Criterion-rs cannot.

Reply:
Interesting. I'd be a bit more skeptical. 

They claim that lai is good for noisy environments because they count the number of instructions. This is true in the general case, but since Iroha has a time-based consensus algorithm (and our tests frequently show instability if not enough performance is available), this kind of instability will add to the number of instructions that get executed. 

We should have benchmarks, but I think we'd need a dedicated test bench.

Reply:
Indeed, the instruction count will increase during timer countdowns